### PR TITLE
Add missing ES6 rules to ESLint config.

### PR DESCRIPTION
**Pull request by simono**
`arrow-body-style` (8.2) and `prefer-template` (6.4) are described in
the Style Guide.

### CODE REVIEW

File: packages/eslint-config-airbnb/rules/es6.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ module.exports = {
     'jsx': true
   },
   'rules': {
+    // enforces no braces where they can be omitted
+    'arrow-body-style': [2, 'as-needed'],
```
Comment (ljharb) [inline packages/eslint-config-airbnb/rules/es6.js]: could we link to http://eslint.org/docs/rules/arrow-body-style in a comment?

File: packages/eslint-config-airbnb/rules/es6.js [hunk 2]
```diff
@@ -50,6 +52,8 @@ module.exports = {
     'prefer-spread': 0,
     // suggest using Reflect methods where applicable
     'prefer-reflect': 0,
+    // suggest using template literals instead of string concatenation
+    'prefer-template': 2,
```
Comment (ljharb) [inline packages/eslint-config-airbnb/rules/es6.js]: could we link to http://eslint.org/docs/rules/prefer-template in a comment?

**Comment by ljharb** [comment]
Thanks, this looks great pending 2 comments :-)

**Comment by simono** [comment]
I've added the links.

Right now the linting for eslint-config-airbnb shows 148 errors. Would you accept a PR which fixes those?

**Comment by ljharb** [comment]
Thanks! i'm not sure what linting you're talking about though.

**Comment by simono** [comment]
Running `npm run lint` in `packages/eslint-config-airbnb` results in 148 errors.

**Comment by ljharb** [comment]
good lord, why isn't that being run as part of tests?? thanks, i'll fix that right now

**Comment by ljharb** [comment]
Thanks, fixed in ab58d3b68eff9fd45db9bc8c78b99e61016e39f9

**Pull request merged**